experiments/run_experiment.py

```diff
@@ -15,13 +15,20 @@
 # Experiment configuration
 
 DEVICES = [
-    habitat.Device.P4000,
-    habitat.Device.P100,
-    habitat.Device.V100,
-    habitat.Device.A100,
-    habitat.Device.T4,
-    habitat.Device.RTX2070,
+    # habitat.Device.P4000,
```

Comment:
You can remove the commented devices

Reply:
Oh good catch. I actually didn't mean to remove it.